### PR TITLE
{tools}[GCCcore/8.3.0, GCCcore/9.3.0] axel v2.17.9

### PR DESCRIPTION
**Pull request by verdurin**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
sms - Linux centos linux 7.6.1810, x86_64, AMD EPYC 7601 32-Core Processor (zen), Python 2.7.5
See https://gist.github.com/ee5f8fe212efae1d6bab19a288acf34d for a full test report.

File: easybuild/easyconfigs/a/axel/axel-2.17.9-GCCcore-8.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+]
+
+dependencies = [('OpenSSL', '1.1.1d')]
+
+sanity_check_paths = {
+    'files': ["bin/axel"],
+    'dirs': [""],
+}
+
+sanity_check_commands = [True, ('%(name)s', '--version')]
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/a/axel/axel-2.17.9-GCCcore-8.3.0.eb:30]: I see that this format, `[True, ...]`, is also used in the `gzip` easyconfigs, but why?

File: easybuild/easyconfigs/a/axel/axel-2.17.9-GCCcore-8.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+sources = [SOURCE_TAR_BZ2]
+checksums = ['d50dfbc59cb04fa70fb8d414579259c3eefe2a87aab52611309feeec9acb851a']
+
+builddependencies = [
+    ('binutils', '2.32'),
+    ('pkg-config', '0.29.2'),
+    ('gettext', '0.20.1'),
+]
+
+dependencies = [('OpenSSL', '1.1.1d')]
```
[the remark this answers is not in the document]
Comment (smoors) [reply easybuild/easyconfigs/a/axel/axel-2.17.9-GCCcore-8.3.0.eb:23]: @verdurin the problem is that the EB-installed `pkg-config` does not look in the system directories, and thus it cannot find the system `openssl`.
you can avoid this as follows:
```suggestion
osdependencies = [('openssl-devel', 'libssl-dev', 'libopenssl-devel')]

configopts = 'SSL_LIBS="-lssl -lcrypto" SSL_CFLAGS=" "'
```

File: easybuild/easyconfigs/a/axel/axel-2.17.9-GCCcore-8.3.0.eb (hunk 1 above)
Comment (smoors) [reply easybuild/easyconfigs/a/axel/axel-2.17.9-GCCcore-8.3.0.eb:30]: @migueldiascosta `True` is equivalent to `%(name)s -h` here.

File: easybuild/easyconfigs/a/axel/axel-2.17.9-GCCcore-8.3.0.eb (hunk 2 above)
[the remark this answers is not in the document]
Comment (boegel) [reply easybuild/easyconfigs/a/axel/axel-2.17.9-GCCcore-8.3.0.eb:23]: This should be changed to the `OpenSSL` wrapper (and the `pkg-config` issue should no longer be relevant thanks to https://github.com/easybuilders/easybuild-easyblocks/pull/2549)

File: easybuild/easyconfigs/a/axel/axel-2.17.9-GCCcore-8.3.0.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/a/axel/axel-2.17.9-GCCcore-8.3.0.eb:30]: I would make that explicit:

```suggestion
sanity_check_commands = [
    "axel -h",
    "axel --version",
]

File: easybuild/easyconfigs/a/axel/axel-2.17.9-GCCcore-8.3.0.eb (hunk 2 above)
[the remark this answers is not in the document]
Comment (boegel) [reply easybuild/easyconfigs/a/axel/axel-2.17.9-GCCcore-8.3.0.eb:23]:
```suggestion
dependencies = [
    ('OpenSSL', '1.1', '', SYSTEM),
]

File: easybuild/easyconfigs/a/axel/axel-2.17.9-GCCcore-9.3.0.eb [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+sources = [SOURCE_TAR_BZ2]
+checksums = ['d50dfbc59cb04fa70fb8d414579259c3eefe2a87aab52611309feeec9acb851a']
+
+builddependencies = [
+    ('binutils', '2.34'),
+    ('pkg-config', '0.29.2'),
+    ('gettext', '0.20.1'),
+]
+
+dependencies = [('OpenSSL', '1.1.1e')]
```
Comment (boegel) [inline easybuild/easyconfigs/a/axel/axel-2.17.9-GCCcore-9.3.0.eb:23]: dependencies = [
    ('OpenSSL', '1.1', '', SYSTEM),
]

File: easybuild/easyconfigs/a/axel/axel-2.17.9-GCCcore-8.3.0.eb [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+sanity_check_paths = {
+    'files': ["bin/axel"],
+    'dirs': [""],
```
Comment (boegel) [inline easybuild/easyconfigs/a/axel/axel-2.17.9-GCCcore-8.3.0.eb:27]:
```suggestion
sanity_check_paths = {
    'files': ['bin/axel'],
    'dirs': [],

File: easybuild/easyconfigs/a/axel/axel-2.17.9-GCCcore-9.3.0.eb [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+sanity_check_paths = {
+    'files': ["bin/axel"],
+    'dirs': [""],
+}
+
+sanity_check_commands = [True, ('%(name)s', '--version')]
```
Comment (boegel) [inline easybuild/easyconfigs/a/axel/axel-2.17.9-GCCcore-9.3.0.eb:30]:
```suggestion
sanity_check_paths = {
    'files': ['bin/axel'],
    'dirs': [],
}

sanity_check_commands = [
    "axel -h",
    "axel --version",
]

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3169.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/2330709951012007738cfd1a18541b79 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @verdurin!

**Pull request merged**